components/http_request.rst

```diff
@@ -241,7 +260,7 @@ If you want to retrieve the value for the ``vol`` key and assign it to a templat
                     id(player_volume).publish_state(root["vol"]);
                 });
 
-**Note:** don't forget to set the ``id`` for the main ``http_request`` component, to ``http_request_data``.
+**Note:** For this example to work, the main ``http_request`` component needs its ``id`` set to ``http_request_data``.
```

Comment:
This example needs to change to reference `body` instead of `id(http_request_data).get_string()`

Reply:
Haha, I knew I was gonna miss _something._ 😅  Good catch & thanks for fixing! 🍻